app/src/main/res/values/colors.xml

```diff
@@ -70,4 +70,5 @@
     <color name="detail_color">#0995d1</color>
     <color name="red">#ec2e2e</color>
     <color name="blue">#35d9f2</color>
+    <color name="background">#dddddd</color>
```

Comment:
Give a proper name to this color, not just background. The name should be relevant to what is its use. Give something like placeholder_back_color or common name of the color ( if any).